### PR TITLE
Localisation updates from https://translatewiki.net.

### PR DESCRIPTION
**Pull request by translatewiki**
Translation updates

### CODE REVIEW

File: app/src/main/res/values-xmf/strings.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Authors:
+* Narazeni
+-->
+<resources>
+  <string name="file_system_does_not_support_4gb">თქვანი ფაილიშ სისტემა ვაუხუჯანს 4გბ-შე უმოს ფაილეფს</string>
```
Comment (MohitMaliFtechiz) [inline app/src/main/res/values-xmf/strings.xml:6]: I am comparing the translation of the `Georgian` language to `English`, and I have noticed that the translation does not match the content in the `string.xml` file. The translations in this file appear different compared to the English version. However, I have observed that the translations in the other files are similar to our string file.
Screenshot from 2023-07-19 18-01-38
 

Screenshot from 2023-07-19 18-01-49

**Review by MohitMaliFtechiz — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-xmf/strings.xml (hunk 1 above)
Comment (kelson42) [reply app/src/main/res/values-xmf/strings.xml:6]: Please report this upstream, on TW, and lets wait a bit to see if this can be corrected.

Comment (MohitMaliFtechiz) [reply app/src/main/res/values-xmf/strings.xml:6]: @kelson42, I have made the changes on upstream and those changes are come in the latest commit of the translate wiki.

**Review by MohitMaliFtechiz — DISMISSED** [review]
Now it's LGTM.

**Comment by kelson42** [comment]
@MohitMaliFtechiz Sorry, but Liniting is still not good. Do I have been to slow to merge and a new problem as been introduced? Anyway plesae fix upstream on TW if necessary.

**Comment by MohitMaliFtechiz** [comment]
> @MohitMaliFtechiz Sorry, but Liniting is still not good. Do I have been to slow to merge and a new problem as been introduced? Anyway plesae fix upstream on TW if necessary.

The linting error was introduced a few days back after i approved the PR, I am placing a fix for this.

**Comment by MohitMaliFtechiz** [comment]
@kelson42 I have placed a fix for the lint issue on `translate-wiki`, lint issue will fix in the upcoming update of TW.

**Review by MohitMaliFtechiz — APPROVED** [review]
@kelson42 Latest changes come in the latest commit of TW and it fixes the lint error, Now it's LGTM.

**Pull request merged**